graf3d/eve7/src/REveTableProxyBuilder.cxx

```diff
@@ -8,33 +8,54 @@
 
 using namespace ROOT::Experimental;
 
+REveTableProxyBuilder::REveTableProxyBuilder() : REveDataProxyBuilderBase("Table"), fTable(0)
+{
+    fTable = new REveDataTable("ProxyTable");
+}
```

Comment:
That about ownership of `fTable`? When it will be cleaned up?

Reply:
The REveDataTable objects is owned by the proxy builder. The destruct-or on REveElements can't be called directly. I have added the Destroy call in the proxy builder destruct-or.